pkgdb2/lib/__init__.py

```diff
@@ -1921,6 +1922,7 @@ def add_unretire_request(session, pkg_name, pkg_branch, user):
         user=user.username,
         _status='Awaiting Review',
         action='request.unretire',
```

Comment:
You should update the comments in api/admin.py, as that says:
    :kwarg action: restrict the actions to a specific action, options are:
        ``request.branch``, ``request.package``.